src/propTypes.js

```diff
@@ -160,3 +210,7 @@ export const jobStore = PropTypes.shape({
   allIds: PropTypes.arrayOf(PropTypes.number.isRequired).isRequired,
 }).isRequired;
 
+export const modelsLabel = PropTypes.arrayOf(PropTypes.shape({
```

Comment:
there is a prop type with a similar content. can you change the name of them to some generic thing like `selectLabel` and use them everywhere?

Reply:
Yes I can look for it